### ISSUE TITLE
Add O_DIRECT

### ISSUE THREAD

**Issue by tmancill** [jnr/jnr-constants#107]
Is there a reason that O_DIRECT couldn't be added to OpenFlags?  Would you accept a PR for it?

For my specific use case, I'm getting some Java code ready to run on both x86-64 and aarch64, and it's inconvenient that the (currently hard-coded constant) for `O_DIRECT` on x86-64 maps to the value of `O_DIRECTORY` on aarch64.

**Comment by headius** [comment]
Do it! Any flag can be added because all flags can be queried to know whether they are defined on the current platform, and when we regenerate for the other platforms we will pick it up if it exists!

**Comment by headius** [comment]
> regenerate for the other platforms

Or modify by hand, carefully. 😀

**Comment by almson** [comment]
@tmancill Also `O_LARGEFILE` and everything else that's in current [`fcntl.h`](https://github.com/torvalds/linux/blob/master/include/uapi/asm-generic/fcntl.h) so that we don't have to ask again 🥺

**Comment by headius** [comment]
Yeah, add anything standard you see and we'll get them generated and released.
